sbin/adduser_course.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+"""
+Use this script to add a user to courses. Any user added to a course
+will be an instructor.
+"""
+
+import argparse
+import json
+from os import path
+import subprocess
+import sys
+from sqlalchemy import create_engine, MetaData, Table, bindparam, and_
+
+CONFIG_PATH = path.join(path.dirname(path.realpath(__file__)), '..', 'config')
+with open(path.join(CONFIG_PATH, 'database.json')) as open_file:
+    DATABASE_DETAILS = json.load(open_file)
+DATABASE_HOST = DATABASE_DETAILS['database_host']
+DATABASE_USER = DATABASE_DETAILS['database_user']
+DATABASE_PASS = DATABASE_DETAILS['database_password']
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Adds a user to courses')
+
+    parser.add_argument('user_id', help='user_id of the user to create')
+    parser.add_argument('semester', help='semester of the course')
+    parser.add_argument('course', help='title of the course')
+    parser.add_argument('registration_section', help='registration section that the user is added into')
```

Comment:
This was missed from my comment, but let's make this final argument optional with default `None`:
`nargs="?", default=None`

Feels cleaner than requiring typing null or whatever on the command line to set no registration section.